opencti-platform/opencti-front/src/private/components/nav/TopMenuAnalysis.tsx

```diff
@@ -54,25 +48,25 @@ const TopMenuAnalysis = () => {
             {t('Reports')}
           </Button>
         )}
-        {!useIsHiddenEntity('Grouping') && (
```

Comment:
We remove grouping from analysis ?

Reply:
I don't know how come I deleted these lines. No. Removing "Grouping" from Analysis is a mistake.
My bad